### PR TITLE
Team activation

### PR DESCRIPTION
**Pull request by solocommand**
Stopgap until we add subscriptions, allows teams to be disabled without removing access roles or IP addresses.

Screen Shot 2019-06-24 at 3 37 21 PM
Screen Shot 2019-06-24 at 3 37 58 PM
Screen Shot 2019-06-24 at 3 37 50 PM

### CODE REVIEW

File: services/graphql/src/graphql/definitions/team.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ type Team {
   id: String! @projection(localField: "_id")
   name: String! @projection
   description: String @projection
+  active: Boolean!
```
Comment (zarathustra323) [inline services/graphql/src/graphql/definitions/team.js]: If you allow this to be a nullable field, it won't break data if the field is missing

File: services/application/src/actions/load-context.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ module.exports = async ({ applicationId, email, ipAddress } = {}) => {
   const accessLevelIds = [];
   const teamQuery = {
     applicationId,
+    active: true,
```
Comment (zarathustra323) [inline services/application/src/actions/load-context.js]: Set this to `{ $ne: false }` and then it will immediately work without data migration

**Review by zarathustra323 — CHANGES_REQUESTED** [review]

**Comment by solocommand** [comment]
@zarathustra323 Added UI stuff and addressed previous comments

File: services/manage/app/templates/components/form-elements/toggle-button.hbs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{{#x-toggle
+  size="medium"
+  theme="light"
+  value=value
+  onLabel=onLabel
+  offLabel=offLabel
+  showLabels=true
+  onToggle=(action (mut value)) as |toggle|
+}}
+  {{toggle.offLabel}}
```
Comment (zarathustra323) [inline services/manage/app/templates/components/form-elements/toggle-button.hbs:10]: The dual-label isn't necessary here. A single "Active" label is fine. Similar to NativeX:
image

**Review by zarathustra323 — CHANGES_REQUESTED** [review]

File: services/manage/app/templates/components/team/modal-form.hbs [hunk 4]
```diff
@@ -79,7 +79,8 @@
       </div>
     {{/modal.body}}
 
-    {{#modal.footer}}
+    {{#modal.footer class="justify-content-between"}}
+      <FormElements::ToggleButton @value={{model.active}} @onLabel="Active" @offLabel="Inactive" />
```
Comment (zarathustra323) [inline services/manage/app/templates/components/team/modal-form.hbs]: Ignore above and just remove the `@offLabel` prop

**Review by zarathustra323 — CHANGES_REQUESTED** [review]

File: services/manage/app/templates/components/team/modal-form.hbs [hunk 5]
```diff
@@ -79,7 +79,8 @@
       </div>
     {{/modal.body}}
 
-    {{#modal.footer}}
+    {{#modal.footer class="justify-content-between"}}
+      <FormElements::ToggleButton @value={{model.active}} @onLabel="Active" />
```
Comment (zarathustra323) [inline services/manage/app/templates/components/team/modal-form.hbs]: While technically it doesn't matter, the new Ember docs want to use `this` when referencing a value in the containing component... as such `@value={{this.model.active}}` - i'm trying hard to switch to this style (though I forget at times)

File: services/manage/app/templates/components/team/status.hbs [hunk 6]
```diff
@@ -0,0 +1 @@
+Status: <span class={{ labelClass }}>{{ label }}</span>
```
Comment (zarathustra323) [inline services/manage/app/templates/components/team/status.hbs]: Remove spaces

**Review by zarathustra323 — CHANGES_REQUESTED** [review]
In addition to these, can the toggle style itself be adjusted to use the bootstrap `$primary` color (e.g. blue instead of the default green)?

**Comment by solocommand** [comment]
Updated screenshots
Screen Shot 2019-06-25 at 3 14 18 PM
Screen Shot 2019-06-25 at 3 14 13 PM

**Review by zarathustra323 — APPROVED** [review]

**Pull request merged**